variables.tf

```diff
@@ -3,14 +3,8 @@ variable "cf_org_name" {
   type        = string
 }
 
-variable "cf_app_domain" {
```

Comment:
Do we sill have the flexibility to select a different domain here? Or is it fixed based on the region?

Reply:
We still need an internal identifier if the prefix is not provided as input, otherwise we will get regional conflicts between customers. It might not make sense to export the "naked" value if we are exporting all relevant endpoints. I expect some points releases over the next few days to address this.

I'm also creating a `thanos-on-hsdp` example project which will consume this new version. This should make the initial conversation with customers who are interested in consuming this much easier.

Reply:
Fixed to the base region for now